### DISCUSSION TITLE
AR Bug: can't load it twice

### DISCUSSION THREAD

**Discussion by jasmin-yc** [google/model-viewer#3392] [Q&A]
I can open and view the AR. When I close it and try to open it again, the error message "Object could not be loaded" appears. If I view file1 in the AR, then file2, then file1 again, it works without problems, only if I want to open the same usdz file twice in a row
![IMG_6C864119837B-1](https://user-images.githubusercontent.com/94001351/165505560-4e268ffb-aaf1-4896-8825-d8c039eb56df.jpeg)
.

**Comment by elalish** [comment]
This is looks like a Quick Look bug. Any thoughts from Apple's side @grorg? You can also file a bug on WebKit.

**Reply by jasmin-yc** [reply]
any updates on this problem?

**Comment by grorg** [comment]
I was unable to reproduce this. I can open the same model multiple times in a row (using the astronaut). Is there a particular model you're having issues with?

**Reply by jasmin-yc** [reply]
Hi, yes, we'd like to use our own model. we tested with the astronaut too. the problem is with our file or our code, we don't know. you can access our model here: https://www.yellowcamper.ch/vorlagen/ubersicht-gestaltungsblocke Password: Burgdorf3400
It is this model here:
<img width="1252" alt="image" src="https://user-images.githubusercontent.com/94001351/166410832-9f0a15d5-4533-4d2a-acad-514c051ae66f.png">

**Reply by jasmin-yc** [reply]
we tried multiple browsers and multiple devices, sometimes you can open the model, sometimes not. it is just one possibility that you can not open the modal a second time, maybe the problem is a different. i evaluated our test and it stand out that this could be the problem.
